src/metabase/api/card.clj

```diff
@@ -52,6 +52,8 @@
    [metabase.util :as u]
    [metabase.util.date-2 :as u.date]
    [metabase.util.i18n :refer [trs tru]]
+   [metabase.util.malli :as mu]
```

Comment:
not really related to the purpose of this PR but since we're switch to malli and I saw some place we could use it.